crates/api_models/src/payment_methods.rs

```diff
@@ -429,6 +429,10 @@ pub struct PaymentMethodListResponse {
     ]
     ))]
     pub payment_methods: Vec<ResponsePaymentMethodsEnabled>,
+    /// Value indicating if the current payment is a mandate payment
+    #[schema(example = "new_mandate_txn")]
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub mandate_payment: Option<payments::MandateTxnType>,
```

Comment:
But the response will be ambigious, consider someone consuming the api, would have to make his decision by checking whether the field is present or not, instead if we always send this field and it's null in case the value is not present, this would make it uniform.

Reply:
That makes sense, but that could also indicate that there is something related to mandate everytime, which is information not always required to have with the list API. It should be very specific to it's purpose and only indicate mandate when required.

Reply:
Yes, but it will be populated only if the value is present right, in other cases it will simple be null. It will also provide more clarity that such a field will now be returned from the api and user has to make the changes accordingly. I personally feel that we do not need the skip_serialization_option here. There are a lot of places where we send optional values, take for example `PaymentsResponse`, if we skip serializing some field because it is not applicable for certain kind of payment, the response struct will be different in different scenarios which would lead to non uniformity and difficult for the consumers of the api